### ISSUE TITLE
What does this have to do with roku?

### ISSUE THREAD

**Issue by ghost** [JoseOneTime/couchtuner#1]
The repository contains various python files... doesn't roku use brightscript for its applications? I don't see how this can work on roku.

**Comment by JoseOneTime** [comment]
You're completely correct. This code was just for scraping ct to create/update the xml file that the  Roku app used. If I still have the brightscript files somewhere, I'll add them when I find them.

**Comment by ghost** [comment]
That would be sweet. As far as I know no one else has created a couchtuner channel for roku.
